crates/rpc/bridge/src/lib.rs

```diff
@@ -15,3 +21,39 @@ pub trait StrataBridgeControlApi {
     #[method(name = "uptime")]
     async fn get_uptime(&self) -> RpcResult<u64>;
 }
+
+#[cfg_attr(not(feature = "client"), rpc(server, namespace = "stratabridge"))]
+#[cfg_attr(feature = "client", rpc(server, client, namespace = "stratabridge"))]
+pub trait StrataBridgeMonitoringApi {
+    /// Get all bridge operator IDs
+    #[method(name = "bridgeOperators")]
+    async fn get_bridge_operators(&self) -> RpcResult<Vec<OperatorIdx>>;
+
+    /// Query operator status (Online/Offline)
+    #[method(name = "operatorStatus")]
+    async fn get_operator_status(&self, operator_idx: OperatorIdx) -> RpcResult<RpcOperatorStatus>;
+
+    /// Get deposit details using the deposit request outpoint
+    #[method(name = "depositInfo")]
+    async fn get_deposit_info(
+        &self,
+        deposit_request_outpoint: OutPoint,
+    ) -> RpcResult<RpcDepositInfo>;
+
+    /// Get bridge duties assigned to an operator
+    #[method(name = "bridgeDuties")]
+    async fn get_bridge_duties(&self, operator_id: u32) -> RpcResult<Vec<BridgeDuty>>;
```

Comment:
This should be split into two
```suggestion
    /// Get bridge duties assigned to an operator by his [`XOnlyPublicKey`].
    #[method(name = "bridgeDuties")]
    async fn get_bridge_duties_by_operator_id(&self, operator_pk: XOnlyPublicKey) -> RpcResult<Vec<BridgeDuty>>;
    
    /// Get bridge duties assigned to an operator by [`OperatorIdx`].
    #[method(name = "bridgeDutiesById")]
    async fn get_bridge_duties_by_operator_id(&self, operator_id: u32) -> RpcResult<Vec<BridgeDuty>>;
```